code/game/gamemodes/colonialmarines/whiskey_outpost.dm

```diff
@@ -404,28 +404,60 @@
 								/obj/item/explosive/grenade/high_explosive,
 								/obj/item/storage/box/nade_box
 								)
-			if(36 to 40) // Junk
+			if(41 to 44)
 				crate = new /obj/structure/closet/crate/ammo(T)
 				spawnitems = list(
 									/obj/item/attachable/heavy_barrel,
 									/obj/item/attachable/heavy_barrel,
 									/obj/item/attachable/heavy_barrel,
 									/obj/item/attachable/heavy_barrel)
 
-			if(40 to 48)//Weapon + supply beacon drop. 6%
+			if(45 to 50)//Weapon + supply beacon drop. 5%
 				crate = new /obj/structure/closet/crate/ammo(T)
 				spawnitems = list(/obj/item/device/whiskey_supply_beacon,
 								/obj/item/device/whiskey_supply_beacon,
 								/obj/item/device/whiskey_supply_beacon,
 								/obj/item/device/whiskey_supply_beacon)
 
-			if(49 to 50)//Rare weapons. Around 4%
+			if(51 to 57)//Rare weapons. Around 6%
 				crate = new /obj/structure/closet/crate/ammo(T)
 				spawnitems = list(/obj/effect/landmark/wo_supplies/ammo/box/rare/m41aap,
 								/obj/effect/landmark/wo_supplies/ammo/box/rare/m41aapmag,
 								/obj/effect/landmark/wo_supplies/ammo/box/rare/m41aextend,
 								/obj/effect/landmark/wo_supplies/ammo/box/rare/smgap,
 								/obj/effect/landmark/wo_supplies/ammo/box/rare/smgextend)
+
+			if(58 to 65) // Sandbags kit
+				crate = new /obj/structure/closet/crate(T)
+				choosemax = rand(3,6)
+				spawnitems = list(/obj/item/tool/shovel/etool,
+								/obj/item/stack/sandbags_empty/half,
+								/obj/item/stack/sandbags_empty/half,
+								/obj/item/stack/sandbags_empty/half)
+
+			if(66 to 75) // Mortar shells. Pew Pew!
+				crate = new /obj/structure/closet/crate/secure/mortar_ammo(T)
+				choosemax = rand(6,10)
+				spawnitems = list(/obj/item/mortar_shell/he,
```

Comment:
this one feels a bit extraneous to me. Mortar crew already gets hundreds of shells of all the types (except frag, but they're terrible)

Reply:
Could add loose bullet crates or AP mags to make it all more meaningful too..

Reply:
> Could add loose bullet crates or AP mags to make it all more meaningful too..

That works